packages/storybook/cxl-ui/cxl-vaadin-accordion/cxl-playbook-accordion.story.js

```diff
@@ -2,19 +2,121 @@ import { html } from 'lit-html';
+// };
+//
+// const setDelay = () => {
+//   clearTimeout();
+//   tmpTimeoutId = window.setTimeout(() => hideAll(), 2000);
+// };
+//
+// const isLayoutWide = () => document.querySelector('cxl-app-layout').wide;
+//
+// const onload = () => {
+//   window.addEventListener('DOMContentLoaded', () => {
+//     document.querySelectorAll('cxl-playbook-accordion').forEach((el) => {
+//       el.addEventListener('mouseleave', () => {
+//         setDelay();
+//       });
+//     });
+//
+//     document.querySelectorAll('vaadin-accordion-panel').forEach((el) => {
+//       el.addEventListener('mouseenter', (evt) => {
+//         if (!isLayoutWide()) {
+//           return;
+//         }
+//         setDelay();
+//
+//         const stepId = evt.target.getAttribute('data-step-id');
+//         const commentIcon = document.querySelector(
+//           `header [icon="vaadin:comment"][data-step-id="${stepId}"]`
+//         );
+//
+//         hideAll();
+//
+//         if (commentIcon) {
+//           commentIcon.classList.remove('hidden');
+//         }
+//       });
+//
+//       el.addEventListener('mouseleave', () => {
+//         if (!isLayoutWide()) {
+//           return;
+//         }
+//         setDelay();
+//       });
+//
+//       el.querySelectorAll('iron-icon').forEach((icon) => {
+//         icon.addEventListener('mouseenter', () => {
+//           clearTimeout();
+//         });
+//
+//         icon.addEventListener('mouseleave', () => {
+//           setDelay();
+//         });
+//
+//         icon.addEventListener('click', (evt) => {
+//           // here should go the comment popup handler click
+//           evt.stopImmediatePropagation();
+//         });
+//       });
+//     });
+//   });
+// };
+
 export const CXLPlaybookAccordion = () => html`
+  <style>
+    .hidden {
+      visibility: hidden;
+    }
+    vaadin-accordion-panel {
+      position: relative;
+    }
+    vaadin-accordion-panel iron-icon.floating {
+      position: absolute;
+      top: calc(50% - 10px);
+      right: calc(var(--lumo-space-l) * -2);
+    }
+    vaadin-accordion-panel iron-icon.floating:hover {
+      cursor: pointer;
+    }
+  </style>
+
   <cxl-playbook-accordion id="playbook-3895" class="plural" opened="0">
     ${playbookStepData.map(
       (el) => html`
-        <vaadin-accordion-panel class="playbook-step" theme="cxl-playbook-accordion reverse">
+        <vaadin-accordion-panel
+          class="playbook-step"
+          theme="cxl-playbook-accordion reverse"
+          data-step-id="${el.id}"
+        >
           <header class="entry-header" slot="summary">
             <h3 class="entry-title no-anchor" itemprop="headline">
               <vaadin-checkbox value="${el.id}"></vaadin-checkbox>
               ${el.id}. ${unsafeHTML(el.title.rendered)}
             </h3>
+            <iron-icon
+              icon="vaadin:comment"
+              class="floating hidden"
+              data-step-id="${el.id}"
+            ></iron-icon>
```

Comment:
Drop icon in header + related styles.

Reply:
Yeap, I was just waiting for the confirmation we drop it. Done.